test/OpenTelemetry.Extensions.Hosting.Tests/HostingExtensionsTests.cs

```diff
@@ -125,5 +164,54 @@ public void Dispose()
                 this.Disposed = true;
             }
         }
+
+        internal class TestActivityProcessor : BaseProcessor<Activity>
```

Comment:
Consider reusing the `./test/OpenTelemetry.Tests/Shared/TestActivityProcessor.cs`.

Reply:
yes. thats the plan.